visidata/loaders/pcap.py

```diff
@@ -300,21 +300,21 @@ class PcapSheet(Sheet):
     columns = [
         ColumnAttr('timestamp', type=date, fmtstr="%H:%M:%S.%f"),
         Column('transport', type=get_transport, width=5),
-        Column('srcmanuf', getter=lambda col,row: manuf(macaddr(row.src))),
-        Column('srchost', getter=lambda col,row: row.srchost),
+        Column('srcmanuf', type=str, getter=lambda col,row: manuf(macaddr(row.src))),
+        Column('srchost', type=str, getter=lambda col,row: row.srchost),
         Column('srcport', type=int, getter=lambda col,row: get_port(row, 'sport')),
-        Column('dstmanuf', getter=lambda col,row: manuf(macaddr(row.dst))),
-        Column('dsthost', getter=lambda col,row: row.dsthost),
+        Column('dstmanuf', type=str, getter=lambda col,row: manuf(macaddr(row.dst))),
+        Column('dsthost', type=str, getter=lambda col,row: row.dsthost),
         Column('dstport', type=int, getter=lambda col,row: get_port(row, 'dport')),
         ColumnAttr('ether_proto', 'type', type=lambda v: protocols['ethernet'].get(v), width=0),
         ColumnAttr('tcp_flags', 'ip.tcp.flags', type=FlagGetter('tcp'), helpstr="TCP Flags"),
-        Column('service', getter=lambda col,row: getService(getTuple(row))),
+        Column('service', type=str, getter=lambda col,row: getService(getTuple(row))),
         ColumnAttr('data', type=vlen),
-        ColumnAttr('ip.len', type=int),
+        ColumnAttr('ip.len', 'ip_len', type=int),
```

Comment:
Seems like these should be reversed (such that `ip_len` is the name of the column, and `ip.len` is the deep attr to get).

Reply:
Correct, will fix that.